src/domain/collaboration/new-callout/CreateCallout/CalloutFormContributionSettings.tsx

```
@@ -0,0 +1,124 @@
+import BlockIcon from '@mui/icons-material/Block';
+import AttachFileOutlinedIcon from '@mui/icons-material/AttachFileOutlined';
+import calloutIcons from '../../callout/utils/calloutIcons';
+import { CalloutContributionType, CalloutType } from '@/core/apollo/generated/graphql-schema';
+import ContributionSettingsDialog from './ContributionSettingsDialog/ContributionSettingsDialog';
+import ContributionsSettingsLink from './ContributionSettingsDialog/ContributionsSettingsLink';
+import ContributionsSettingsPost from './ContributionSettingsDialog/ContributionsSettingsPost';
+import ContributionsSettingsWhiteboard from './ContributionSettingsDialog/ContributionsSettingsWhiteboard';
+
+interface CalloutFormContributionSettingsProps {}
+
+const CalloutFormContributionSettings = ({}: CalloutFormContributionSettingsProps) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)